tests/config/require-prettier.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+const prettier = require("prettier");
+
+module.exports = prettier;
```

Comment:
what is this file for?

Reply:
the tests can run using `prettier` or `prettier/standalone` as the engine.